content/docs/languages/csharp/dotnet.md

```diff
@@ -2,11 +2,12 @@
 title: gRPC for .NET
 ---
 
-For an introduction to gRPC for .NET ([grpc-dotnet][]) and a basic tutorial, see
-the following pages:
+For an introduction to gRPC for .NET ([grpc-dotnet][]), a basic tutorial, and
+API reference documentation, see the following pages:
```

Comment:
This is redundant. You're just repeating the items listed below.

Reply:
Yeah, I was feeling that too. Fixed.